db/schema.rb

```diff
@@ -10,24 +10,24 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170319180638) do
+ActiveRecord::Schema.define(version: 20170321150557) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "hstore"
 
   create_table "activities", force: :cascade do |t|
     t.integer  "team_id"
-    t.string   "kind",         limit: 255
```

Comment:
I'm not sure why the `limit: 255` for most fields was removed in the schema file — I'm guessing this might have happened automatically with your newly added migration.I don't know how badly we need those limits on fields, but since this is not relevant to the PR, could you change all of them back to their previous state? (i.e. add `limit: 255` to the fields again where it's been removed). Thanks :)